{
  getMockEndpoint("mock:result").expectedMessageCount(0);
  try {
    template.sendBody("direct:d","World");
    fail("Should have thrown exception");
  }
 catch (  CamelExecutionException e) {
    ExpressionEvaluationException cause=assertIsInstanceOf(ExpressionEvaluationException.class,e.getCause());
    assertTrue(cause.getCause().getMessage().startsWith("Unknown function: xxx at location 0"));
  }
  assertMockEndpointsSatisfied();
}
